### ISSUE TITLE
Make possible to ask for additional text also for default prompts

### ISSUE THREAD

**Issue by micz** [micz/ThunderAI#135]
Make possible to ask for additional text also for default prompts.
Default prompts can be enabled and disabled only. Let the user change also che "Ask for additional text checkbox".

**Comment by twigandolive** [comment]
I saw your reply in the other thread, so I'm following up here. I looked at the idea of custom prompts, but I found that I would need many of the various replies I'd be doing. What it comes down to is the nuances of each client/email. Of course, I do "Reply to this," wait for the text to generate and then add the modified response. It's less of a custom prompt and more of "normally we would respond X in this situation, but here we need to respond Y," which is a different response altogether. It's okay, but having a custom reply would save a step.

As an alternative, what about a kill switch of sorts so that when the default prompts are replied to and queried, instead of waiting for the response to generate, have a command (keyboard shortcut) to kill the reply and then allow for additional input? Thoughts?
